static/js/common/upload-base.js

```diff
@@ -6,7 +6,15 @@
         return results.errors;
     }
 
-    var settings = {'filetypes': [], 'getErrors': getErrors, 'cancel': $()};
+    function textSize(bytes) {
+        // Based on code by Cary Dunn (http://bit.ly/d8qbWc).
```

Comment:
Not that it matters, but that url is a 404.  Also, I despair that JS doesn't have a built-in method for this.